### PR TITLE
Use custom GPU binaries if custom path provided

### PR DESCRIPTION
**Pull request by yashanand1910**
## Describe your changes
If custom path for the GPU binaries is provided via the config items:

- `gpu_controller_path` (or env var `CEDANA_GPU_CONTROLLER_PATH`)

- `gpu_shared_lib_path` (or env var `CEDANA_GPU_SHARED_LIB_PATH`)

Then, it should not download these binaries into the default locations but use the binaries provided on these paths. This will help in testing/benchmarking.

## Issue ticket number 
CED-390

## Checklist before requesting a review
- [x] Have I performed a self-review?
- [ ] Have I added regression or unit tests (where it makes sense) for my changes?
- [x] Have I updated the README if changes have resulted in it being out of date?
- [x]  Have I checked to ensure my PR only introduces the changes it's intended to? E.g. No extraneous formatting changes.
- [ ] Is this a breaking change? If yes, please describe areas affected and request reviews from developer stakeholders.

### CODE REVIEW

File: cmd/exec.go [hunk 1]
```diff
@@ -59,7 +58,6 @@ var execTaskCmd = &cobra.Command{
 		gpuEnabled, _ := cmd.Flags().GetBool(gpuEnabledFlag)
 		if gpuEnabled {
 			logger.Info().Msgf("starting task w/ gpu enabled")
-			executable = fmt.Sprintf("LD_PRELOAD=%s %s", utils.GpuSharedLibPath, executable)
```
Comment (nravic) [inline cmd/exec.go:62]: I think we should leave this in so customers don't have to manually LD_PRELOAD their gpu tasks. Does this not work on your machine @yashanand1910 ?

**Review by nravic — COMMENTED** [review]
looks good! added a small comment

File: cmd/exec.go (hunk 1 above)
Comment (yashanand1910) [reply cmd/exec.go:62]: Oh! This was simply moved to the daemon. Check `process.go`, where we start the controller. Just thought to keep it at one place.

Comment (nravic) [reply cmd/exec.go:62]: Ah missed that - thank you! Yeah better in process than cmd/exec.

**Review by nravic — APPROVED** [review]
lgtm!

**Pull request merged**